docs/api.md

```diff
@@ -4,18 +4,21 @@ auro-carousel displays a group of elements in a scrollable container.
 
 ## Properties
 
-| Property         | Attribute        | Type      | Default | Description                                      |
-|------------------|------------------|-----------|---------|--------------------------------------------------|
-| `displayArrows`  | `displayArrows`  | `Boolean` |         | Forces left and right navigation to stick in DOM regardless of content width |
-| `label`          | `label`          | `String`  |         | The accessible name for the carousel. Logs a console warning if not set. |
-| `scrollDistance` | `scrollDistance` | `Number`  | 300     | How many pixels to scroll the carousel when the shoulder buttons are triggered. |
+| Property                 | Attribute                | Type      | Default | Description                                      |
+|--------------------------|--------------------------|-----------|---------|--------------------------------------------------|
+| `centerSelectedOnRender` | `centerSelectedOnRender` | `string`  |         |                                                  |
```

Comment:
Add the following to the head of the auro-carousel.js file.

```
* @attr {String} centerSelected -Apply to outer auro-carousel element to automatically center the selected node on UI render. ```

Reply:
Sure.